### PR TITLE
ST110 - CaseField

### PR DESCRIPTION
**Pull request by rshunim**
## Related Issues
relates: https://jira-dc.paloaltonetworks.com/browse/CIAC-7809

## Description
Added CaseField object of Pydantic for ST110 validation.

### CODE REVIEW

File: demisto_sdk/commands/content_graph/strict_objects/case_field.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    item_version: Optional[str] = Field(None, alias="itemVersion")
+    propagation_labels: Optional[Any] = Field(None, alias="propagationLabels")
+    to_server_version: Optional[str] = Field(None, alias="toServerVersion")
+    open_ended: Optional[bool] = Field(None, alias="openEnded")
+    template: Optional[str] = None
+    marketplaces: Literal[MarketplaceVersions.MarketplaceV2] = Field(
+        default_factory=list
+    )
+    aliases: Optional[List[StrictAliases]] = Field(None, alias="Aliases")
+    x2_fields: Optional[str] = None
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/strict_objects/case_field.py:74]: we got this in content?

File: demisto_sdk/commands/content_graph/strict_objects/case_field.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    associated_to_all: Optional[bool] = Field(None, alias="associatedToAll")
+    unmapped: Optional[bool] = None
+    content: Optional[bool] = None
+    unsearchable: Optional[bool] = None
+    item_version: Optional[str] = Field(None, alias="itemVersion")
+    propagation_labels: Optional[Any] = Field(None, alias="propagationLabels")
+    to_server_version: Optional[str] = Field(None, alias="toServerVersion")
+    open_ended: Optional[bool] = Field(None, alias="openEnded")
+    template: Optional[str] = None
+    marketplaces: Literal[MarketplaceVersions.MarketplaceV2] = Field(
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/strict_objects/case_field.py:70]: is mpv2 the only allowed value here? (`Literal`)

File: demisto_sdk/commands/content_graph/strict_objects/case_field.py (hunk 1 above)
Comment (rshunim) [reply demisto_sdk/commands/content_graph/strict_objects/case_field.py:74]: There are no caseFields in Content at all so I didn't remove it, WDTY?

Comment (dorschw) [reply demisto_sdk/commands/content_graph/strict_objects/case_field.py:74]: remove it

File: demisto_sdk/commands/content_graph/strict_objects/case_field.py (hunk 2 above)
Comment (rshunim) [reply demisto_sdk/commands/content_graph/strict_objects/case_field.py:70]: image

**Review by dorschw — APPROVED** [review]

**Pull request merged**